src/main/scala/cromwell/engine/backend/jes/JesRuntimeInfo.scala

```diff
@@ -9,7 +9,7 @@ object JesRuntimeInfo {
   def apply(commandLine: String, call: Call): JesRuntimeInfo = {
     val dockerImage = call.docker.getOrElse(throw new IllegalArgumentException("FIXME: WHAT TO DO?"))
```

Comment:
Haha I forgot about this - another one of those things which was never supposed to make it to the real jes backend release.

I think this is a trickier question than it seems as the docker is still an option. Sure, we "know" that it's a Some but we don't *know*. Could do something w/ a subtype of runtimeattributes which has a non-optional docker but that's just an off the cuff thought so dunno exactly how that would work yet.